control/content/app.services.js

```diff
@@ -472,14 +471,14 @@
                         userMessage: `List sample coupons for a new [Optics Shop]`,
                         maxRecords: 5,
                         systemMessage:
-                            "listImage is an 1080x720 image URL related to title and the list type, use source.unsplash.com for images, URL should not have premium_photo or source.unsplash.com/random.",
+                            "listImage is an image URL related to title and the list type, Use https://app.buildfire.com/api/stockImages/?topic={title}&imageType=medium, A maximum of 2 comma-separated title can be used for each URL",
```

Comment:
listImage is an image URL related to the title or the list type, Use https://app.buildfire.com/api/stockImages/?topic={topic}&imageType=medium, a maximum of 2 comma-separated topics can be used for each URL.